site/_data/i18n/common.yml

```
@@ -93,6 +93,10 @@ codelab:
   en: 'Codelab'
   es: 'Codelab'
 
+example:
+  en: 'Example'
+  es: 'Ejemplo'
```

```suggestion
en: 'Example'
es: 'Ejemplo'
ja: '例'
ko: '예'
pt: 'Exemplo'
ru: 'Пример'
zh: '例子'
```

Single words are safe to be auto-translated.